pulser/devices/_mock_device.py

```diff
@@ -47,5 +47,11 @@
             "mw_global",
             Microwave.Global(1000, 200, clock_period=1, min_duration=1),
         ),
+        (
+            "mw_local",
+            Microwave.Local(
+                1000, 200, 0, max_targets=2000, clock_period=1, min_duration=1
+            ),
+        ),
```

Comment:
At first I did it because I thought that the samples dictionary in `Simulation` was based on the existing channels, but that's not the case so you're right